skrf/calibration/tests/test_calibration.py

```diff
@@ -70,6 +68,47 @@ def test_determine_reflect(self):
 
         [ self.assertEqual(k,l) for k,l in zip(self.r, r_found)]
 
+    def test_determine_reflect_matched_thru_and_line_ideal_reflect(self):
+        freq = rf.F(.25, .7, 21, unit = 'GHz')
+        medium=Coaxial.from_attenuation_VF(freq, att = 3.0, VF = .69)
```

Comment:
Spaces around `=`.

Reply:
Sorry about all the small whitespace issues. Hopefully with https://github.com/scikit-rf/scikit-rf/pull/892, these will all be handled automatically.